eng/native/functions.cmake

```diff
@@ -421,46 +423,51 @@ function(strip_symbols targetName outputFilename)
         TARGET ${targetName}
         POST_BUILD
         VERBATIM
+        COMMAND sh -c "echo Stripping symbols from $(basename '${strip_source_file}') into $(basename '${strip_destination_file}')"
```

Comment:
I moved this since it was confusing to have the "Stripping" message printed _after_ we're done stripping